quorum/values.yaml

```diff
@@ -7,6 +7,11 @@ geth:
   raftPort: 50401
   verbosity: 3
   gethParams: --permissioned \ --nodiscover \ --nat=none \ --unlock 0 \ --emitcheckpoints \ --rpccorsdomain '*' \ --rpcvhosts '*' \
+epirus:
+  enabled: true
+  port: 3000
```

Comment:
Does this port really need to be configurable? Other ports like the one for the API are hard-coded.

Reply:
Removed the port for now, if the need arises we can make everything adjustable.